cocos/base/ccTypes.h

```diff
@@ -566,7 +568,10 @@ struct CC_DLL FontDefinition
     FontShadow            _shadow;
     /// font stroke
     FontStroke            _stroke;
-
+    /// enable text wrap
+    bool                  _enableWrap;
+    /// enable shrink font size
+    int                  _overflow;
```

Comment:
Why it's an integer value? If it has to be integer, it's better to add comments to describe the meaning of integer value.